src/DynamoCore/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Greg" version="1.3.7408.23353" targetFramework="net47" />
-  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
-  <package id="RestSharp" version="105.2.3" targetFramework="net47" />
+  <package id="Greg" version="1.3.7408.23353" targetFramework="net48" />
```

Comment:
I guess we'll want to update package manager client to .net 4.8 as well at some point.